betatest/templates/betatest/application.html

```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+{% load compress %}
+
+{% block title %}Register for Beta{% endblock title %}
+
+{% block css %}
+{% compress css %}
+<link href="{% static "scss/betatest.scss" %}" type="text/x-scss" rel="stylesheet"/>
+{% endcompress %}
+{% endblock css %}
+
+{% block head_js %}
+<script src="{% static "djng/js/django-angular.min.js" %}"></script>
+{% endblock head_js %}
+
+{% block content %}
+<div class="container content" ng-app="BetaTestApp">
+    <div class="row">
+        <div class="large-12 columns">
+            <h1>Register for Beta</h1>
+            <p>This is the application form for the beta of the self-service
+            hosting offered by OpenCraft. For enterprise or institutional
+            offerings, contact us.</p>
```

Comment:
Can you make "contact us" a link to contact@opencraft.com ?

(Just lurking quickly too : ) The PR looks neat! Glad to see the integration of forms with angular : ) )